tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -591,7 +591,7 @@ test_i2c_ina230: ina230@5e {
 	reg = <0x5e>;
 	config = <0>;
 	current-lsb-microamps = <1000>;
-	rshunt-milliohms = <0>;
+	rshunt-micro-ohms = <1000>;
```

Comment:
Looks like the undetected run-time divide by zero error in the test case turned into a detected compile time error.

Reply:
yep, hence the fix of actually setting it.